README.md

```diff
@@ -1 +1,47 @@
-# Test-To-Do
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name: TodoList Testing
+
+> Description the project: With the design of this project the user can add todo list and can set it as completed or not. it's easy to use and simple ;)
+
+
+## Built With
+
+- Major languages
+> Html amd JavaScript
+
+
```

Comment:
- Please, kindly add a Getting started header and add some instructions on how a user can have your project locally on their machine. 
- How to run your app
- How to run your test
- How to install your project dependencies